community/CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Argo Projects
+
+This document explains the common procedures expected by contributors while submitting code to Argo projects.
+
+## Code of Conduct
+
+Please read and abide by the [CNCF Community Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+
+## General workflow
+
+Once a github issue is accepted and assigned to you, please follow general workflow in order to submit your contribution:
+1. Fork the target repository under your github username.
+2. Create a branch in your forked repository for the changes you are about to make.
+3. Commit your changes in the branch you created in step 2. All commits need to be signed-off. Check the [legal](#legal) section bellow for more details.
+4. Push your commits to your remote fork.
+5. Create a Pull Request from your remote fork pointing to the HEAD branch (usualy `master` branch) of the target repository.
+6. Check the github build and ensure that all checks are green.
```

Comment:
Maybe we should mention the recent changes to GitHub actions, that PRs from first time contributors to a given project require approval before the checks being run (and that it can take a while until someone presses the button).

Reply:
@jannfis Is this something configured for all Argo projects?

Reply:
I think it is, yes. Either it was an organization wide setting, or it was just set as the default by GitHub when they introduced this feature. Some projects may have opted out.